### PR TITLE
Fix hash and install of behringer-x32-edit

### PR DESCRIPTION
**Pull request by alecsammon**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew audit --cask {{cask_file}}` is error-free.
- [x] `brew style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask-drivers/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew install --cask {{cask_file}}` worked successfully.
- [x] `brew uninstall --cask {{cask_file}}` worked successfully.
- [ ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask-drivers/search?q=is%3Aclosed&type=Issues).
- [ ] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

**Comment by alecsammon** [comment]
Looks like the download of this app has changed from a simple zip of the single .app to a folder including the release history as both a .txt and .pdf. This has caused the hash to change.

I have also added the `desc` field - taken from the product description on https://www.behringer.com/product.html?modelCode=P0AWF

Finally I have included `~/Library/.X32-Edit` to the list of locations to zap. This is the place where the application caches local state.

I have checked install, uninstall and zap.

**Comment by alecsammon** [comment]
As an extra point - I am unsure if this should be in `homebrew-cask-drivers` or if it should be moved to `homebrew-cask`.

The X32 is a digital audio mixing desk. This software is a remote control app for the hardware device (over LAN/Wi-Fi) and not a hardware driver.

I believe it is not a driver as it does not allow other software to communicate with the device, and does not provide a local api for other software to consume. Additionally the device is a remote device, not directly connected to the computer, and is fully functional without this software installed.

**Comment by vitorgalvao** [comment]
> This software is a remote control app for the hardware device (over LAN/Wi-Fi) and not a hardware driver.

We [define drivers](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) as:

> Software to make a hardware peripheral recognisable and usable by the system. **If the software is useless without the peripheral, it’s considered a driver.**

Emphasis added. It belong in this repo.

File: Casks/behringer-x32-edit.rb [hunk 1]
```diff
@@ -1,20 +1,22 @@
 cask "behringer-x32-edit" do
   # NOTE: "32" is not a version number, but an intrinsic part of the product name
   version "4.2"
-  sha256 "d94ff3b039f3732da7c8b2d930c9f75c97b0012ae4adaca99d456c842abde057"
+  sha256 "69297f08ff205016f4490f891a6f50b623d2a50752019459d7d91a4f809c1520"
 
   # mediadl.musictribe.com/download/software/behringer/ was verified as official when first introduced to the cask
   url "https://mediadl.musictribe.com/download/software/behringer/X32/X32-Edit_MAC_#{version}.zip"
   name "Behringer X32-Edit"
+  desc "Edit Behringer Digital Mixers via LAN/Wi-Fi"
```
Comment (vitorgalvao) [inline Casks/behringer-x32-edit.rb:9]: Powerful software application

“Powerful” is marketing bluff; [software application](https://mobile.twitter.com/ines_durao/status/1331738867062480903) is superfluous (it’s what we carry).

**Review by vitorgalvao — APPROVED** [review]

**Comment by alecsammon** [comment]
@vitorgalvao thank you - have learnt so much more about submitting patches to homebrew now - will ensure that any future PR's I submit follow the guidelines.

And thank you for all the work you do as part of the homebrew team!

**Pull request merged**